Massdrop/react-menu
# Maintain focus on menu when clicking/tapping on disabled option.

Asana ticket: https://app.asana.com/0/33626641634273/155232389681340

##### Description of Functionality Added or Modified
react-menu uses the onBlur event to decide when to close an open menu. When clicking on a disabled option like the "Show More" link in the notifications menu, the body of the page becomes focused so future clicks no longer triggers the onBlur even that react-menu is listening for.

Can't use react-menu/MenuOption `onSelect` prop for this because that automatically closes the menu.

##### Steps to Test/Reproduce

1. Make sure you have at least 10 notifications. If you test with an account that can create Threads, a fast way to generate notifications is to use another account to repeatedly follow/unfollow your main account. Each follow will generate a notification.
2. On iOS device or simulator, load your local app.
3. Tap the Notifications icon in the nav and tap the Show More link at the bottom of the Notifications menu.
4. Tap the Communities icon, Search icon, or User icon in the nav. The Notifications menu show hide itself.
5. Also test this on an Android device or simulator.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/massdrop/react-menu/5)
